### PR TITLE
Added ability to pass array of plugins

### PR DESCRIPTION
**Pull request by AdriVanHoudt**

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
@csrl any news on merging or issues?

**Comment by csrl** [comment]
Lines 290 - 400 of test/index.js should not be nested within another test.
It also seems overly redundant to duplicate all the tests between the two configurations.  Please trim it down to just adding test that cover the new functionality.

File: lib/index.js [hunk 1]
```diff
@@ -123,11 +123,23 @@ exports.compose = function (manifest /*, [options], callback */) {
...
+                var pluginObject = manifest.plugins[i];
+                var keys = Object.keys(pluginObject);
 
-            var plugin = internals.parsePlugin(name, manifest.plugins[name], options.relativeTo);
-            plugins = plugins.concat(plugin);
-        });
+                Hoek.assert(keys.length === 1, 'Invalid plugin config');
+
+                var plugin = internals.parsePlugin(keys[0], pluginObject[keys[0]], options.relativeTo);
+                plugins = plugins.concat(plugin);
```
Comment (Marsup) [inline lib/index.js]: Why not push instead of creating a new array at each turn ?

Comment (AdriVanHoudt) [inline lib/index.js]: I copied from the original but in this case it would idd do the same, is there a reason not to use concat?

Comment (csrl) [inline lib/index.js]: The 'plugin' value may or may not be an array.  If it is an array, it should be concatenated, not pushed.  The use of concat allows for either.

**Comment by AdriVanHoudt** [comment]
also might be good to enable container on travis to make the builds faster, took 15min to wait on travis results

**Pull request merged**